mobile/app/home/profile.tsx

```diff
@@ -56,13 +56,13 @@ export default function Page() {
   };
 
   const onPressNotification = async () => {
-    if (!activeAccount) {
-      console.log("No active account");
-      return;
+    if (!account) {
+      throw new Error("No active account");
     }
+
     setLoading(true);
     try {
-      const address_bech32 = await gnonative.addressToBech32(activeAccount?.address);
+      const address_bech32 = await gnonative.addressToBech32(account.address);
```

Comment:
We need to pay attention to the difference between a byte array address and a bech32 string. Previously, `activeAccount` was the response from `gnonative.getActiveAccount()` where it has a byte array address. But `useAppSelector(selectAccount)` returns a `User`, right? This has the bech32.